Allow to use `../` in view path (#282)
yiisoft/view: CHANGELOG.md

## 12.0.1 under development

> - Enh #282: Allow to use the name of the view starting with `../` to join the parent directory containing the view 

Reviewer: ```suggestion
- Enh #282: Allow using `../` in the name of the view to refer to parent directory of the the directory containing the view 
```

>   currently being rendered (@vjik)

Reviewer: What's the use-case for it?

Author: When view placed in parent directory, e. g. `../_header.php`.

Reviewer: What if it's two levels up?

Author: It's work. I just added such test.

Reviewer: While it might be handy, it poses a potential security issue if view name is obtained from user's input, i.e. user may use `../../etc/passwd` or something alike to read files.

Author: Seems, it's was always available. For example, `./../../etc/passed/`.

Should we somehow prevent this (I don't have ideas how)? Or note adding to render() method will be enough?

Reviewer: A note should be enough, I think.

Author: Done
